src/savegame_json.cpp

```diff
@@ -1665,7 +1666,9 @@ void vehicle_part::deserialize(JsonIn &jsin)
         { "external_water_tank", { "external_tank", "water_clean" } },
         { "water_tank_barrel", { "tank_barrel", "water_clean" } },
 
-        { "napalm_tank", { "tank", "napalm" } }
+        { "napalm_tank", { "tank", "napalm" } },
+
+        { "hydrogen_tank", { "tank", "plasma" } }
```

Comment:
Lets go with `none` then. The odds of anyone having plasma in a tank are defacto zero given how exceedingly rare it is and the other uses being much more valuable (plasma weapons and cvd machines)